data-prepper-pipeline-parser/src/main/java/org/opensearch/dataprepper/pipeline/parser/README.md

```diff
@@ -0,0 +1,75 @@
+## Pipeline Configuration Transformation
```

Comment:
Please move this to the `/docs` directory of the project. And rename to `pipeline_configuration_transformation.md`.

Reply:
addressed in the latest pr